api/spec/requests/spree/api/orders_controller_spec.rb

```diff
@@ -815,6 +815,38 @@ module Spree
           expect(response.status).to eq 201
           expect(json_response["user_id"]).to eq(user.id)
         end
+
+        context 'with payment', js: true do
```

Comment:
I think `js: true` is not needed for these specs

Reply:
done here https://github.com/solidusio/solidus/pull/3066/files#diff-72468e52a59aedfe8a3443597f1a6f3aR836